Havit.Blazor.Components.Web.Bootstrap/Calendar/HxCalendar.razor.cs

```diff
@@ -105,6 +106,19 @@ static HxCalendar()
 	/// </summary>
 	[Parameter] public bool KeyboardNavigation { get; set; } = true;
 
+	// Set during SetParameterSetAsync to make it optional
+	[Inject] public TimeProvider TimeProviderFromServices { get; set; }
+	/// <summary>
+	/// TimeProvider is resolved in the following order:<br />
+	///		1. TimeProvider from this parameter <br />
+	///		2. Settings TimeProvider (configurable from <see cref="HxCalendar.Settings"/>)<br />
+	///		3. Defaults TimeProvider (configurable from <see cref="HxCalendar.Defaults"/>)<br />
+	///		4. TimeProvider from DependencyInjection<br />
```

Comment:
Consider using `ol` + `li` HTML elements for formatting this block (should work with IntelliSense but please verify).

Reply:
Unfortunately IntelliSense uses this markup:
```
/// <list type="number">
///     <item>TimeProvider from this parameter</item>
/// </list>
```
 but the documentation server doesn't support that.

Reply:
Oops. Leave it as it is then. Thank you.